src/main/resources/schemas/single_justice_procedure_press.json

```diff
@@ -208,8 +208,6 @@
                                               "description": "Individual Details",
                                               "type": "object",
                                               "required": [
-                                                "individualForenames",
-                                                "individualSurname",
                                                 "dateOfBirth",
                                                 "age",
                                                 "address"
```

Comment:
I think address should be made optional here too, now that all address fields are optional. Not sure what impact this will have on channel management though?